### DISCUSSION TITLE
is opt_interactive possible on pkgdown site?

### DISCUSSION THREAD

**Discussion by rkb965** [rstudio/gt#1933] [Q&A]
Hi, all!

I have a gt table in a package vignette that benefits massively from `opt_interactive`. My vignette renders as expected when I build my package, but when I try to build a pkgdown site, the contents of my table disappear. My title and footnotes still render, but I have no rows. 

I don't really know where to start for debugging this so would love any insight!  

My vague guesses from googling involve something about custom css/js and potentially modifying the yaml of my .qmd vignette (including the `pkgdown::as_is`), but I am rather lost. Thank you!

**Comment by olivroy** [comment]
Umm, we are able to [include](https://gt.rstudio.com/articles/gt-interactive.html) interactive tables without more work with a [Quarto vignette](https://pkgdown.r-lib.org/articles/quarto.html).

It is possible that pkgdown >= 2.1.0 broke something for regular Rmd vignettes. I saw that in reactable, they are using pkgdown 2.0.9 for their site. https://github.com/glin/reactable/commit/b756218f7d1404cdd9af0f437f7cc4e5d395052f

**Reply by rkb965** [reply]
@olivroy Oh thank you, that is a relief that this should work naively.

But I am indeed using quarto vignettes. Do you have any suggestions where I should start for troubleshooting? I am on gt 0.11.1, pkgdown 2.1.1, quarto software version 1.4.11, quarto package version 1.4.4, devtools version 2.4.5.9000.

Again, the table appears as expected when I render the quarto file outside of pkgdown. But `build_site()` results in an html with the table captions and footnotes but no body.

My DESCRIPTION file has VignetteBuilder: quarto and my vignette header has:

```
vignette: >
  %\VignetteIndexEntry{My functions}
  %\VignetteEngine{quarto::html}
  %\VignetteEncoding{UTF-8}
```

This renders with the interactive table as expected:

```
> quarto::quarto_render("vignettes/my-article.qmd")


processing file: my-article.qmd
                                                                                                             
output file: my-article.knit.md

pandoc 
  to: html
  output-file: my-article.html
  standalone: true
  section-divs: true
  html-math-method: mathjax
  wrap: none
  default-image-extension: png
  toc: true
  
metadata
  document-css: false
  link-citations: true
  date-format: long
  lang: en
  title: My functions
  vignette: |
    %\VignetteIndexEntry{My functions} %\VignetteEngine{quarto::html} %\VignetteEncoding{UTF-8}
  
Output created: my-article.html
```

Thank you so much for any thoughts!

**Reply by olivroy** [reply]
quarto 1.4.11 is outdated. You should update Quarto. New released version is Quarto 1.6.40

**Reply by rkb965** [reply]
Thank you for noting that, @olivroy ! I had attempted to point to a newer quarto but I guess successfully. I confirmed that my quarto vignette is now using quarto 1.6.39.

One thing that I noticed in the browser developer tools is that the table in the locally-rendered vignette has <class="reactable html-widget html-widget-static-bound"> but when I render through pkgdown::build_site(), the table has <class="reactable html-widget"> only. Is that informative?
